md/release-notes.md

```diff
@@ -190,3 +190,54 @@ This used to store Legacy v6 forms. When migrating a production environment to 7
 
 ### Debug action in Studio
 Debug, the option to run a diagram without its connectors, is not supported anymore, as its value proved to be too small.
+
+## Bug fixes
+
+#### Fixes in Documentation
```

Comment:
❓ some in this list are not marked as resolved in 7.8 in jira.
I don't know how you manage this list